CHANGELOG.md

```diff
@@ -0,0 +1,155 @@
+# Changelog
+
+This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.4.9]() - 2019-10-23
+
+### Changed
+
+- Only print out number of configs and rules when running with verbose flag
+- Match let and const to var in javascript:
+    ```
+    var $F = "hello"
+    ```
+    will now match any of the following expressions:
+    ```javascript
+    var foo = "hello";
+    let bar = "hello";
+    const baz = "hello";
+    ```
+
+### Added
+
+- Print out --dump-ast
+- Print out version with `--version`
+- Allow ... in arrays
+    ```
+    [..., 1]
+    ```
+    will now match
+    ```
+    [3, 2, 1]
+    ```
+- Support Metavariable match on keyword arguments in python:
+    ```
+    foo(..., $K=$B, ...)
+    ```
+    will now match
+    ```
+    foo(1, 2, bar=baz, 3)
+    ```
+- Support python fstring variable substitution
```

Comment:
I'm not sure this works https://sgrep.live/ROq
https://github.com/returntocorp/sgrep/issues/326
https://github.com/returntocorp/sgrep/pull/347

Reply:
Apparently, when you do have multiple statements in the file that matches the pattern, the sgrep.live is misleading and show the entire list of statements as matching.
@aryx  told me that
> it’s because of the way sequence matching work
> it would just say the whole seqence of stmts match
> put each test in a separate function
See https://sgrep.live/eG8, which puts each of those cases in a function to have independent matching test( which the test files in #347 precisely does)

Reply:
Oh thanks!